### PR TITLE
Add SwapMath.sol UniSwap contract

### LINKED ISSUE #2: Benchmark from Uniswap math
https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/SqrtPriceMath.sol
https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/FullMath.sol
https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/TickMath.sol

### PR DESCRIPTION
**Pull request by rodiazet**
Closes #2

### CODE REVIEW

File: src/main/swap_math.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  transaction:
+    to: '0xbe7c43a580000000000000000000000000000001'
+    data:
+    - :label computeSwapStep 0x100d3f74000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000010146dd68287f35899f20af670000000000000000000000000000000000000000000000001bc16d674ec800000000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000000000000258
```
Comment (chfast) [inline src/main/swap_math.yml]: They should have different names. Can be `computeSwapStep_1`.

**Review by chfast — CHANGES_REQUESTED** [review]
There seems to be something wrong with these because every input consumes exactly 424 gas. This looks live very low number for 1000+ instruction code.

File: src/main/swap_math.yml (hunk 1 above)
Comment (rodiazet) [reply src/main/swap_math.yml]: Done. Please check if this also fixed general problem mentioned above.

**Comment by rodiazet** [comment]
> 

Done. I've been using wrong byte code. @chfast please verify that it works now.

File: src/main/swap_math.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  transaction:
+    to: '0xbe7c43a580000000000000000000000000000001'
+    data:
+    - :label computeSwapStep_1 0x100d3f74000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000010146dd68287f35899f20af670000000000000000000000000000000000000000000000001bc16d674ec800000000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000000000000258
```
Comment (chfast) [inline src/main/swap_math.yml]: Can you add a comment which is the matching unit test?

Comment (rodiazet) [reply src/main/swap_math.yml]: Done

**Review by chfast — APPROVED** [review]

**Pull request merged**